src/pages/stop/stop.component.ts

```diff
@@ -184,8 +184,8 @@ export class StopComponent {
       sExact: moment(sdt).format('LT'),
       eExact: moment(edt).format('LT'),
       // ex: '6 minutes'
-      sRelativeNoPrefix: moment(sdt).fromNow(true),
-      eRelativeNoPrefix: moment(edt).fromNow(true)
+      sRelativeNoPrefix: moment(sdt).fromNow(),
```

Comment:
Minor nit-pick: the `NoPrefix`, I believe, refers to the fact that there _isn't_ an 'in' or an 'ago'.

Reply:
Yup, just the variable naming should be changed.